backend/.env.dev

```
@@ -7,7 +7,7 @@ APP_SECRET=your_app_secret_here
 APP_URL=http://localhost:8080
 
 # Database configuration
-DATABASE_URL="mysql://cbz_user:cbz_password@database:3306/cbz_reader?serverVersion=8.0"
+DATABASE_URL="mysql://${MYSQL_USER:-cbz_user}:${MYSQL_PASSWORD:-cbz_password}@database:3306/${MYSQL_DATABASE:-cbz_reader}?serverVersion=8.0.32&charset=utf8mb4"
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The DATABASE_URL key is not assigned properly

(SubstitutionKey)